docs/design.md

```diff
@@ -424,29 +424,39 @@ supplied to provide an upper bound on timestamps for already-committed
 data (`ε` is the maximum clock skew). As the transaction progresses, any
 data read which have timestamps greater than `t` but less than `t+ε`
 cause the transaction to abort and retry with the conflicting timestamp
-t<sub>c</sub>, where t<sub>c</sub> \> t. The maximum timestamp `t+ε` remains the same.
-Time spent retrying because of reading recently committed data has an
-upper bound of `ε`. In fact, this is further optimized: upon restarting,
-the transaction not only takes into account the timestamp of the future
-value, but the timestamp of the node at the time of the uncertain read.
-The larger of those two timestamps (typically, the latter) is used to
-bump up the read timestamp, and additionally the node is marked as
-“certain”. This means that for future reads to that node within the
-transaction, we can set `MaxTimestamp = Read Timestamp` (and hence avoid
-further uncertainty restarts). Correctness follows from the fact that we
+t<sub>c</sub>, where t<sub>c</sub> \> t. The maximum timestamp `t+ε` remains
+the same.
+
+We apply another optimization to avoid further restarts caused
+by uncertainty. Upon restarting, the transaction not only takes
+into account t<sub>c<sub>, but the timestamp of the node at the time
+of the uncertain read t<>node<>. The larger of those two timestamps:
+max(t<sub>c<sub>, t<sub>node<sub>) (more likely the latter) is used
```

Comment:
`more likely the latter` should move before the max because it doesn't refer to it.